nginx/default.template

```diff
@@ -15,9 +15,25 @@ server {
   # Long timeouts are necessary for extended split reports
   proxy_read_timeout 300s;
 
+  location ~ /districtmapping/plan/\d*/unlockedgeometries/ {
+    client_max_body_size 20m;
+    client_body_buffer_size 1m;
+
+    # Larger buffers are necessary for responses containing
```

Comment:
Nitpick, but I think we want to communicate that there are more buffers, not larger ones.